### PR TITLE
Cicd

### PR DESCRIPTION
**Pull request by kat-hi**

### CODE REVIEW

**Comment by kat-hi** [comment]
@troeger please review... I've got some hints for discussion:
- ingress hostname
- FE CICD still happens in the frontend-repository. we could have both builds in one repo but afaik this would always trigger both builds even if only one repo is updated. 
- could you add the TEST section for kubeportal to the workflow? I am not sure with that.

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  run:
+    working-directory: kubeportal-api
+
+jobs:
+  TEST-DEV-API:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.5
```
Comment (troeger) [inline .github/workflows/main.yaml]: At least Python 3.6 is needed

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.5'
+      - name: install dependencies
+        run: |
+          apt-get update && apt-get install -y gcc make libc-dev musl-dev libffi-dev libssl-dev libpcre3-dev
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install --no-cache-dir -r requirements-dev.txt; fi
```
Comment (troeger) [inline .github/workflows/main.yaml]: Please use requirements-prod.txt

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      - name: run Unittest
+        run: ./manage.py test --configuration=Development
```
Comment (troeger) [inline .github/workflows/main.yaml:35]: This is correct

File: .github/workflows/main.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+jobs:
+  TEST-DEV-API:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.5
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.5'
```
Comment (troeger) [inline .github/workflows/main.yaml]: At least Python 3.6 is needed.

**Review by troeger — CHANGES_REQUESTED** [review]

**Comment by kat-hi** [comment]
@troeger RUN unittest still fails. could you have a look?

**Comment by troeger** [comment]
Please pull the latest fe-api here, it contains a fix for the failing single test. I was not able to do that:

shaw:kubeportal-integration ptroeger$ make pull-api
git subtree pull --prefix kubeportal-api https://github.com/kubeportal/kubeportal.git fe-api --squash
fatal: Konnte Remote-Referenz fe-api nicht finden.

**Pull request closed without merging**